deployment/CP03ISSM_Deploy.csv

```diff
@@ -86,86 +86,86 @@ AT-31,,AR-04,,CP03ISSM-RID27-03-CTDBPC000,3,1,2015-10-21T17:44:00,2016-05-14T13:
+AR-08,,AR-18,,CP03ISSM-RIC21-00-CPMENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGCON-NCPM02-00004,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID26-00-DCLENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGCON-NDCL26-00023,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID26-04-VELPTA000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-VELPTA-12234,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID26-06-PHSEND000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-PHSEND-P0069,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID26-07-NUTNRB000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-NUTNRB-00269,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID26-08-SPKIRB000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-SPKIRB-00290,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID27-00-DCLENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGCON-NDCL27-00004,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID27-01-OPTAAD000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-OPTAAD-00257,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID27-02-FLORTD000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-FLORTD-00993,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID27-03-CTDBPC000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-CTDBPC-50109,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-RID27-04-DOSTAD000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-DOSTAD-00514,40.36192,-70.87827,,7,92,
+AR-08,,AR-18,,CP03ISSM-SBC11-00-CPMENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGCON-BCPM01-00008,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD11-00-DCLENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGCON-BDCL11-00008,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD11-01-MOPAK0000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-MOPAK0-43257,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD11-02-HYDGN0000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-HYDGN0-41917,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD11-06-METBKA000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-METLGR-00012,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD12-00-DCLENG000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGCON-BDCL12-00008,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD12-03-HYDGN0000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-HYDGN0-42316,40.36192,-70.87827,,0,92,
+AR-08,,AR-18,,CP03ISSM-SBD12-04-PCO2AA000,5,1,2016-10-11T13:39:00,2017-06-15T15:56:00,CGMCP-03ISSM-00005,,CGINS-PCO2AA-35262,40.36192,-70.87827,,0,92,
```

Comment:
Reviewed corrections to deploy CUID, sensor UIDs and deployment depths. Looks good.